.github/workflows/go.yml

```diff
@@ -45,6 +45,10 @@ jobs:
         key: golangci-lint-v1.51.2
       if: ${{ matrix.command == 'lint' }}
 
+    - name: Install linter
+      run: go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.51.2
+      if: ${{ matrix.command == 'lint' }}
+
     - name: Build the weaver binary
```

Comment:
Oh sorry, I meant to suggest removing this 'Build the weaver binary' step. I think the 'Install linter' step is needed.

Reply:
Oh I see. I think it works.